collaboration/sharing-data.md

```diff
@@ -137,6 +137,6 @@ The Fiddler Everywhere client informs the recipients about shared resources (inc
 
     * Confirm the opening of the resource.
 
-    * The shared resources load in the respective section (for example, a shared session loads in the **Snapshots** list, a rule in the **Rules Queue**, and a request in the **Requests** collection). If the shared resource is a snapshot of sessions, its content immediately loads in a new tab.
+    * The shared resources load in the respective section (for example, a shared session loads in the **Snapshots** list, a rule in the **Rules** queue, and a request in the **Requests** collection). If the shared resource is a snapshot of sessions, its content immediately loads in a new tab.
```

Comment:
'a snapshot of sessions' - this one confuses me. maybe we can just keep **If the shared resource is a snapshot**?
Maybe the fact that we just renamed sessions to snapshots is overwhelming for me.

Reply:
revamped a bit